base/special/trig.jl

```diff
@@ -1254,7 +1254,11 @@ Simultaneously compute the sine and cosine of `x`, where `x` is in degrees.
 !!! compat "Julia 1.3"
     This function requires at least Julia 1.3.
 """
-function sincosd(x::Real)
+sincosd(x::Number) = (sind(x), cosd(x))
+# It turns out that calling these functions separately yields better
+# performance than considering each case and calling `sincos_kernel`.
+
+function deleteremainder()
```

Comment:
This `deleteremainder` function makes no sense.

Reply:
@vtjnash add `deleteremainder`, he said,

> I think that removing the constraint isn't quite enough, since the checks in the function are not necessarily correct for other types either.

Would you care to comment, @vtjnash?

Reply:
I think he meant to "delete the remainder of this function", not to literally put it into a function called `deleteremainder()`.

Reply:
I guess so, too.

Reply:
Yes.